src/components/Pages/Search/Search.js

```diff
@@ -1,7 +1,102 @@
+import {
+  TextField,
+  ThemeProvider,
+  Button,
+  createMuiTheme,
+  Tabs,
+  Tab,
+} from "@material-ui/core";
+import { useState, useEffect } from "react";
+import SearchIcon from "@material-ui/icons/Search";
+import CustomPagination from "../../Pagination/CustomPagination";
+import SingleContent from "../../SingleContent/SingleContent";
+import "./Search.css";
+
 const Search = () => {
+  const [type, setType] = useState(0);
```

Comment:
Couldn't this be either "tv" or "movie"? That way you wouldn't have to do `type ? "tv" : "movie"` later on in your API call (line 34). Changing this would have an effect on your `Tabs` though...

Reply:
I changed it, but without the selection the Searchfield doesn't find anything.